package.json

```diff
@@ -2,7 +2,8 @@
 	"scripts": {
 		"start": "ng serve",
 		"build": "ng build lib && gulp compileStyles && npm run build:schematics && gulp copy",
-		"build:ci": "ng build lib && ng build dev-app --aot && npm run build:schematics",
```

Comment:
warum fallen die Schematics wieder raus?

Reply:
Tun sie nicht. Ich brauche nur dieses script nicht mehr, in dem alle builds nacheinander ausgeführt werden. Stattdessen werden nun alle builds in der action parallel ausgeführt. Siehe https://github.com/plentymarkets/terra-components/pull/1223/files#diff-e9f950f17198d3d5e3122a44230a09b9R27-R32